packages/core/src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -213,6 +234,25 @@ function ProductDetails({
                 isValidating={isValidating}
                 taxesConfiguration={taxesConfiguration}
               />
+
+              {skuMatrix?.shouldDisplaySKUMatrix &&
+                Object.keys(slugsMap).length > 1 && (
+                  <>
+                    <div data-fs-product-details-settings-separator>Or</div>
```

Comment:
I think the value `Or` should come from the `hCMS` to avoid being locked just with this value (and in English only).